pkg/common/constants.go

```diff
@@ -17,9 +17,9 @@ package common
 
 const (
 	// LabelAnnotationPrefix is the prefix of every labels and annotations added by the controller.
-	LabelAnnotationPrefix = "data.fluid.io/"
-	// The format is data.fluid.io/storage-{runtime_type}-{data_set_name}
-	LabelAnnotationStorageCapacityPrefix = LabelAnnotationPrefix + "storage-"
+	LabelAnnotationPrefix = "fluid.io/"
+	// The format is data.fluid.io/s-{runtime_type}-{data_set_name}, s means storage
```

Comment:
```suggestion
	// The format is fluid.io/s-{runtime_type}-{data_set_name}, s means storage
```
Remove "data." in the comment